app/src/main/java/it/ilker/repeatcard/ui/quiz/QuizViewModel.kt

```diff
@@ -1,16 +1,17 @@
 package it.ilker.repeatcard.ui.quiz
 
 import android.content.Context
-import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import it.ilker.repeatcard.db.FlashcardDatabase
 import it.ilker.repeatcard.db.flashcard.FlashcardRepository
 import it.ilker.repeatcard.models.question.Question
 import it.ilker.repeatcard.models.quizresult.QuizResult
 import it.ilker.repeatcard.ui.util.exhaustive
-import java.util.UUID
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.launch
+import java.util.*
```

Comment:
We should maintain the old import instead of * import.

Reply:
Fixed!